.circleci/scripts/install_npm.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+mkdir /test
```

Comment:
nit: This is probably fine, but it might be marginally safer to use something like `cd $(mktemp -d)` here to create a more hermetic test directory.